### PR TITLE
revision 1

### PR DESCRIPTION
**Pull request by Nisaba**

### CODE REVIEW

**Comment by SwissBitcoinPay** [comment]
At the "Payment method" selection page, is there a way to change this logo to something more generic like "Bitcoin Lightning" ?
image

Replace it with like:
image

File: SwissBitcoinPaySettings.cs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    /// </summary>
+    public class SwissBitcoinPaySettings : ISettings
+    {
+        /// <summary>
+        /// The url of your the Swiss Bitcoin Pay API
+        /// </summary>
+        public string ApiUrl { get; set; }
+
+        /// <summary>
+        /// Your Swiss Bitcoin Pay API Key given in your Swiss Bitcoin Pay dashbord
```
Comment (SwissBitcoinPay) [inline SwissBitcoinPaySettings.cs]: Tiny typo :)

```suggestion
        /// Your Swiss Bitcoin Pay API Key given in your Swiss Bitcoin Pay dashboard

```

File: SwissBitcoinPaySettings.cs [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        /// </summary>
+        public string ApiKey { get; set; }
+
+        /// <summary>
+        /// The Api Secret value set in your Swiss Bitcoin Pay dashbord
```
Comment (SwissBitcoinPay) [inline SwissBitcoinPaySettings.cs]: Tiny typo :)

```suggestion
        /// The Api Secret value set in your Swiss Bitcoin Pay dashboard

```

File: plugin.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+﻿{
+  "Group": "Payment methods",
+  "FriendlyName": "Swiss Bitcoin Pay",
+  "SystemName": "Payments.SwissBitcoinPay",
+  "Version": "1.01",
+  "SupportedVersions": [ "4.60" ],
+  "Author": "Nisaba Solutions",
+  "DisplayOrder": 1,
+  "FileName": "Nop.Plugin.Payments.SwissBitcoinPay.dll",
+  "Description": "This plugin enables Bitcoin payment with Swiss Bitcoin Pay"
```
Comment (SwissBitcoinPay) [inline plugin.json]: More specific description

```suggestion
  "Description": "This plugin enable Bitcoin Onchain/Lightning payments with Swiss Bitcoin Pay"

```

**Review by SwissBitcoinPay — COMMENTED** [review]
Some typos and description improvement

File: SwissBitcoinPaySettings.cs (hunk 2 above)
Comment (Nisaba) [reply SwissBitcoinPaySettings.cs]: ok done

File: plugin.json (hunk 3 above)
Comment (Nisaba) [reply plugin.json]: ok done

File: SwissBitcoinPayPaymentProcessor.cs [hunk 4]
```diff
@@ -0,0 +1,283 @@
...
+
+            //locales
+            await _localizationService.AddOrUpdateLocaleResourceAsync(new Dictionary<string, string>
+            {
+                ["Plugins.Payments.SwissBitcoinPay.AdditionalFee"] = "Additional fee",
+                ["Plugins.Payments.SwissBitcoinPay.AdditionalFee.Hint"] = "The additional fee.",
+                ["Plugins.Payments.SwissBitcoinPay.AdditionalFeePercentage"] = "Additional fee. Use percentage",
+                ["Plugins.Payments.SwissBitcoinPay.AdditionalFeePercentage.Hint"] = "Determines whether to apply a percentage additional fee to the order total. If not enabled, a fixed value is used.",
+                ["Plugins.Payments.SwissBitcoinPay.Instructions"] = "Enter your data below to configure this plugin :",
+                ["Plugins.Payments.SwissBitcoinPay.PaymentMethodDescription"] = "Pay in bitcoins with Swiss Bitcoin Pay",
```
Comment (SwissBitcoinPay) [inline SwissBitcoinPayPaymentProcessor.cs]:
```suggestion
                ["Plugins.Payments.SwissBitcoinPay.PaymentMethodDescription"] = "Pay in Bitcoin with Swiss Bitcoin Pay",

```

**Review by SwissBitcoinPay — CHANGES_REQUESTED** [review]
Last typo issue

**Pull request merged**